djangocms_versioning/cms_config.py

```diff
@@ -157,6 +157,16 @@ def on_page_content_unpublish(version):
     page.clear_cache(menu=True)
 
 
+def on_page_content_draft_create(version):
+    page = version.content.page
+    page.clear_cache(menu=True)
+
+
+def on_page_content_archive(version):
```

Comment:
The body of this function is exactly the same as the one above. Is this right?

Reply:
yes it is but I kept it like that if in case we need to add extra kwargs. maybe can have _clear_menu_cache function and call it.